homeassistant/components/notify/__init__.py

```diff
@@ -161,6 +166,14 @@ async def async_setup(
         self._target_service_name_prefix = target_service_name_prefix
         self.registered_targets = {}
 
+        # Load service descriptions from notify/services.yaml
+        integration = await async_get_integration(hass, DOMAIN)
+        services_yaml = integration.file_path / "services.yaml"
+        if services_yaml.exists():
+            self.services_dict = cast(dict, load_yaml(str(services_yaml)))
+        else:
+            self.services_dict = {}
```

Comment:
This is an async context and we don't have to check if it exists either. Tests will fail if it doesn't. 
```suggestion
        self.services_dict = cast(
            dict,
            await hass.async_add_executor_job(load_yaml, str(services_yaml))
        )
```